plugins/source/bitbucket/testdata/config.yml

```diff
@@ -0,0 +1,15 @@
+kind: source
+spec:
+  name: bitbucket
+  version: v1.0.0
+  destinations: [test]
+  path: "bitbucket:latest"
```

Comment:
is it a correct path?

Reply:
It is in this case. This config is used to generated the markdown tables using a locally built docker image called `bitbucket:latest`